nextflow/main.nf

```diff
@@ -0,0 +1,32 @@
+params.read_triplets = "$projectDir/test_data/reads/reads_{1,2,3}*.gz"
+params.outdir = "results"
+
+process UMI_INJECTOR {
+    container 'umi-injector'
+    cpus 2
```

Comment:
I started trying your suggestion but ended up using params to set the resources cpus, memory and time. This is because I realised the uppmax profile of nf-core is actually using those and not the label, I think the label is translated to resources in base configs in the pipelines, for example for rnaseq: https://github.com/nf-core/rnaseq/blob/6e1e448f535ccf34d11cc691bb241cfd6e60a647/conf/base.config#L37

Using params is to make it easy for users to change these accordingly, it's not really standard, but when there is only a single process in the workflow it might make sense?

Reply:
Yes, exactly, it is the base config that sets those.

I agree, your approach to set the resources is more reasonable.